lib/WebService/Braintree/Configuration.pm

```diff
@@ -1,14 +1,96 @@
 package WebService::Braintree::Configuration;
 
+=head1 NAME
+
+WebService::Braintree::Configuration
+
+=head1 PURPOSE
+
+This keeps all configuration information for your WebService::Braintree
+installation.
+
+A singleton of this class is instantiated when you use L<WebService::Braintree>.
+You are intended to set attributes of this class immediately, then the rest of
+the distribution knows what to do.
+
+=cut
 
 use WebService::Braintree::Gateway;
 use Moose;
 
-has merchant_id => (is => 'rw');
+# IS THIS UNUSED? I cannot find reference in the current documentation for Ruby
```

Comment:
[this](https://github.com/braintree/braintree_ruby/blob/master/spec/unit/braintree/configuration_spec.rb#L35) (and similar int he python lib) suggests that it might be an alias used in some circumstances.  My best guess is it's a thing that's part of the braintree/paypal gateway (vzero is using the paypal REST api in places according to my source)